CONTRIBUTING.md

```diff
@@ -15,13 +15,21 @@ When reporting a bug in the navigation SDK itself, please indicate:
 
 ## Building the SDK
 
-To build this SDK, you need Xcode 14 and [Carthage](https://github.com/Carthage/Carthage/):
+To build this SDK, you need Xcode 15 and [Carthage](https://github.com/Carthage/Carthage/):
```

Comment:
Carthage is not required anymore :)

Reply:
As we want to bring back the Example I thought to leave this in. If you think its better to remove it, I'll do

Reply:
Ah ok, so for the Examples we would need Carthage?

Reply:
Good point, we shouldn't need Carthage at all, therefore I've removed this. I think for the Examples it should be enough to add a new Xcode project and add the local Package as dependency.